fitlins/cli/run.py

```diff
@@ -96,10 +96,12 @@ def get_parser():
                         help="use BOLD files with the provided description label")
 
     g_prep = parser.add_argument_group('Options for preprocessing BOLD series')
-    g_prep.add_argument('-s', '--smoothing', action='store', metavar="TYPE:FWHM",
+    g_prep.add_argument('-s', '--smoothing', action='store', metavar="FWHM:LEVEL:TYPE",
                         help="Smooth BOLD series with FWHM mm kernel prior to fitting. "
                              "Valid types: iso (isotropic); "
-                             "e.g. `--smothing iso:5` will use an isotropic 5mm FWHM kernel")
+                             "Defaults: LEVEL='subject' TYPE='iso';"
+                             "e.g. `--smoothing 5:dataset:iso will use an isotropic 5mm"
+                             "FWHM on subject-level maps, before evaluating the dataset level.")
```

Comment:
I had a couple minor comments, in particular the default should be `l1` and the metavar should use square brackets to indicate optional components. I also think we skip around a little bit and want to restructure the help text, so here's the whole thing rewritten:

```Python
    g_prep.add_argument('-s', '--smoothing', action='store', metavar="FWHM[:LEVEL:[TYPE]]",
                        help="Smooth BOLD series with FWHM mm kernel prior to fitting at LEVEL. "
                             "Optional analysis LEVEL (default: l1) may be specified numerically "
                             "(e.g., `l1`) or by name (`run`, `subject`, `session` or `dataset`). "
                             "Optional smoothing TYPE (default: iso) must be one of: `iso` (isotropic). "
                             "e.g., `--smoothing 5:dataset:iso` will perform a 5mm FWHM isotropic "
                             "smoothing on subject-level maps, before evaluating the dataset level.")
```

Feel free to edit for clarity.